mlflow/recipes/utils/metrics.py

```diff
@@ -84,17 +84,21 @@ def _get_error_fn(tmpl: str, use_probability: bool = False, positive_class: Opti
         return lambda predictions, targets: predictions - targets
     if tmpl == "classification/v1":
         if use_probability:
-            import numpy as np
 
-            def logloss(true_label, predicted_proba, eps=1e-15):
-                p = np.clip(predicted_proba, eps, 1 - eps)
+            def error_rate(true_label, predicted_positive_class_proba, eps=1e-15):
```

Comment:
looks like `eps` is no longer being used - can we remove it?

Reply:
Ahhh yes. Good catch. Will remove it